### ISSUE TITLE
Comparing to a control?

### ISSUE THREAD

**Issue by ben-ix** [maximtrp/scikit-posthocs#10]
Hi there,

Thanks for the package! I am using friedman tests paired with the Nemenyi test, and this works very nicely for looking at all pairwise comparisons.

I was wondering if there was a way to compare vs a control method using say the Holm method?
I can see there is a Holm option with both conover and siegel, but I do not believe these compare against a control (correct me if Im wrong).

Thank you

**Comment by ben-ix** [comment]
Yes this is what I meant, okay thanks anyway! This would be a nice addition in the future. By the way @maximtrp, if I use this in a paper, how would you like me to cite the package?

**Comment by maximtrp** [comment]
@ben-ix 
There is no other way to cite it now, but to add a package name when describing statistical analysis. But I'm planning to publish a paper in JOSS in the near future.

**Comment by maximtrp** [comment]
@ben-ix You can [cite](https://github.com/maximtrp/scikit-posthocs#citing) the package now.
